src/patternfly/components/Select/select-toggle-button.hbs

```diff
@@ -3,7 +3,14 @@
     <i class="fas fa-caret-down pf-c-select__toggle-arrow" aria-hidden="true"></i>
   {{/button}}
 {{else}}
-  {{#> button button--modifier="pf-m-plain pf-c-select__toggle-button" button--attribute=(concat 'id="' id '-toggle" aria-haspopup="true" aria-expanded="false" aria-labelledby="' id '-label ' id '-toggle" aria-label="Select"')}}
-    <i class="fas fa-caret-down pf-c-select__toggle-arrow" aria-hidden="true"></i>
-  {{/button}}
+  {{#if select--IsDisabled}}
+    {{#> button button--modifier="pf-m-plain pf-c-select__toggle-button pf-m-disabled" 
```

Comment:
Since this renders as a `<button>` element, we want to apply the `disabled` attribute to the element, not the `.pf-m-disabled` class. We use `disabled` on form elements that support it. We use `.pf-m-disabled` on non-form elements, like `<a class="pf-c-button pf-m-disabled">a link that looks like a button</a>`, since an `<a>` doesn't support the `disabled` attribute.